composer.json

```diff
@@ -28,7 +28,8 @@
     "oat-sa/extension-tao-proctoring" : ">=20.0.0",
     "oat-sa/extension-tao-testqti" : ">=41.0.0",
     "oat-sa/extension-tao-outcome" : ">=13.0.0",
-    "oat-sa/extension-tao-testqti-previewer" : ">=3.0.0"
+    "oat-sa/extension-tao-testqti-previewer" : ">=3.0.0",
+    "oat-sa/extension-tao-outcomeui": "dev-bug/TR-1909/incorect-answers-represented-in-review as 100.0.0"
```

Comment:
Bumping a major version of a dependency is a breaking change too.
IMO, extension-lti-test-review's major version should be bumped as well.

Reply:
this pr already has breaking change commit, also this comment  (https://github.com/oat-sa/extension-lti-test-review/pull/94#issuecomment-979187680), as i understand, means that major version will be bumped by this pr